script/bootstrap

```diff
@@ -9,14 +9,14 @@ fi
 STATUS=0
 
 if ! go version; then
-  echo "You need to install Go 1.2 to build hub" >&2
+  echo "You need to install Go 1.4.1 or higher to build hub" >&2
   STATUS=1
 fi
 
 { ruby --version
   bundle install --path vendor/bundle -j 4
 } || {
-  echo "You need Ruby 2.1 and Bundler to run hub tests" >&2
+  echo "You need Ruby 1.9 or higher and Bundler to run hub tests" >&2
```

Comment:
I'm torn about this. On one hand, our test suite does support Ruby 1.9. On the other hand, I put 2.1 in there because I wanted to suggest to people to install the most recent version if they're going to install Ruby for hub development. Ruby versions older than 2.1.4 have all sorts of compilation issues on newer systems that were fixed in the Ruby build process of newer releases.